bw2data/backends/iotable/backend.py

```diff
@@ -22,8 +22,8 @@ class IOTableBackend(SQLiteBackend):
     backend = "iotable"
     node_class = IOTableActivity
 
-    def write(self, data, process=False, searchable=True):
-        super().write(data, process=False, searchable=searchable)
+    def write(self, data, process=False, searchable=True, check_typos=True):
+        super().write(data, process=process, searchable=searchable, check_typos=check_typos)
```

Comment:
This needs to be `process=False` for `IOTable` - the whole point of this class is to have a different way of creating the processed arrays.

Reply:
@cmutel pardon, took this back with [8b13b65](https://github.com/brightway-lca/brightway2-data/pull/175/commits/8b13b654c2e4004e691007f28f6d4fe3f379ac80)